src/config/navigation.js

```diff
@@ -0,0 +1,40 @@
+import { routesConfiguration } from 'config/routes';
+import IconDashboard from 'components/icon/Dashboard';
+import IconMessages from 'components/icon/Messages';
+import IconMentorships from 'components/icon/Mentorships';
+import IconSettings from 'components/icon/Settings';
+
+const navConfiguration = {
+  tabs: [
+    {
+      name: routesConfiguration.dashboard.name,
+      label: 'Home', // trans key => dashboard.navigation.tab.home
```

Comment:
I don't know how to use the translations here... should we just add the keys and use them on the component? I wanted to have them translated here !